packages/k2-alpine/package.json

```diff
@@ -2,10 +2,7 @@
   "name": "@avalabs/k2-alpine",
   "license": "Limited Ecosystem License",
   "version": "1.0.0",
-  "main": "expo/AppEntry.js",
-  "files": [
-    "./src/index.ts"
-  ],
```

Comment:
do we still need `expo`, `expo-font`, `expo-splash-screen` and `expo-status-bar`?